src/test/java/com/epam/tc/hw5/steps/AssertStep.java

```diff
@@ -0,0 +1,141 @@
+    public void usernamesShouldBeDisplayedUsersTableOnUserTablePage(int numberOfUsers) {
+        softAssert.assertThat(userTablePage.getUsers().size()).isEqualTo(numberOfUsers);
+    }
+
+    @And("{int} Description texts under images should be displayed on Users Table on User Table Page")
+    public void descriptionTextsUnderImagesShouldBeDisplayedUsersTableOnUserTablePage(int numberOfDescriptions) {
+        softAssert.assertThat(userTablePage.getDescriptions().size()).isEqualTo(numberOfDescriptions);
+    }
+
+    @And("{int} checkboxes should be displayed on Users Table on User Table Page")
+    public void checkboxesShouldBeDisplayedUsersTableOnUserTablePage(int numberOfCheckboxes) {
+        softAssert.assertThat(userTablePage.getCheckboxes().size()).isEqualTo(numberOfCheckboxes);
+    }
+
+    @Then("User table should contain following values:")
+    public void userTableShouldContainValues(DataTable dataTable) {
+        List<List<String>> userTable = dataTable.rows(1).cells();
+        List<String> number = new ArrayList<>();
+        List<String> user = new ArrayList<>();
+        List<String> description = new ArrayList<>();
+        for (List<String> element : userTable) {
+            number.add(element.get(0));
+            user.add(element.get(1));
+            description.add(element.get(2));
+        }
+        softAssert.assertThat(userTablePage.getNumberFromTable())
+              .as("Incorrect numbers")
+              .isEqualTo(number);
+        softAssert.assertThat(userTablePage.getUserFromTable())
+              .as("Incorrect users")
+              .isEqualTo(user);
+        softAssert.assertThat(userTablePage.getDescriptionFromTable())
+              .as("Incorrect description")
+              .isEqualTo(description);
+        softAssert.assertAll();
+    }
+
+    @Then("droplist should contain values in column Type for user {string}")
+    public void droplistShouldContainValuesInColumn(String userName, io.cucumber.datatable.DataTable dataTable) {
+        List<List<String>> valuesTable = dataTable.rows(1).cells();
+        List<String> values = new ArrayList<>();
+        for (List<String> element : valuesTable) {
+            values.add(element.get(0));
+        }
+        softAssert.assertThat(userTablePage.getDroplistValues(userName))
+              .as("Incorrect values in dropdown")
+              .isEqualTo(values);
+        softAssert.assertAll();
+    }
+
+    @Then("1 log row has {string} text in log section")
+    public void logRowHasTextInLogSection(String log) {
+        softAssert.assertThat(userTablePage.logsAfterSelectingVipCheckbox(log)).as("Fail").isTrue();
+    }
+
+    private static boolean logsContainsItem(Collection<WebElement> logs, String expectedEntry) {
+        for (var logElement : logs) {
+            if (logElement.getText().contains(expectedEntry)) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
Is it possible to place boolean logsContainsItem method in Different Elements page class?

Reply:
I placed and renamed boolean areLogsContainsItem() method in Different Elements page